test/tests/markers/reporter_point_marker/point_marker_test.i

```diff
@@ -0,0 +1,71 @@
+[Mesh]
```

Comment:
You should be able to simplify this test significantly by not running anything at all. Remove the kernels, variables, BCs and just have:

```
[Problem
  solve = false
[]

[Executioner]
  type = steady
[]
```

And do the refine on initial.